doc/Building.md

```diff
@@ -41,12 +41,12 @@ Install the prerequisites using [Homebrew](https://brew.sh/):
 
 ```
 brew install nasm ragel binutils coreutils libtool autoconf automake cmake makedepend \
-sdl2 lua@5.1 luarocks gettext pkg-config wget
+sdl2 lua@5.1 luarocks gettext pkg-config wget gnu-getopt
```

Comment:
You'll also want to add it here:
https://github.com/koreader/koreader/blob/a90038e4ebc501c1c14e06e5a63b3b007fbf15a8/.github/workflows/build.yml#L25

(and possibly in base just to keep it in sync)

Reply:
Unless I have an old checkout, it's already not in sync in base ;). @pazos?

Reply:
Is it not in sync or just not including junk it doesn't need like getopt?

Reply:
It's not even using the same XCode version AFAICT ^^.

Reply:
Slipped a comment about it in https://github.com/koreader/koreader-base/pull/1257/commits/e47b0a9cbb719fde50595e7ab0fd1ddc0d3d261a

(We're not using kodev in base, so we can do without).